cityscrape-config.sh

```diff
@@ -0,0 +1,16 @@
+# Variables for the cityscrape
+
+# Postgresql Database Settings
+# make sure you have enabled PostGIS
+export DATABASE_HOST='localhost'
+export DATABASE_NAME='city'
+export DATABASE_USER='postgres'
+export DATABASE_PASSWORD=''
+
+# URL housing zip files of open city data
+export SCRAPE_URL='https://github.com/hopcity/cityscrape/'
+
+# Directory in which scripts residek and a subfolder, 'zipFiles' exists
+# for temporary file download and extraction before loading into database
+export BASE_DIR='/home/clay/Downloads/stlCity/'
+
```

Comment:
you can create a variable called something like BASEDIR = `dirname "$0"` and that way the filepath is all referential and bootstraps itself at runtime, thus allowing you to plop the project in anywhere and run it assuming you have directory perms and whatnot...